framework/base/View.php

```diff
@@ -334,11 +334,17 @@ public function renderPhpFile($_file_, $_params_ = [])
      * when some portions of the content (called *dynamic content*) should not be cached.
      * The dynamic content must be returned by some PHP statements.
      * @param string $statements the PHP statements for generating the dynamic content.
+     * @param array $params the parameters (name-value pairs) that will be extracted and made
```

Comment:
@dynasource I've copied the sentence from the other `$params`, e.g. from `render()`. There's no `be` there either.